### PR TITLE
Prep-javascript-objects-exercise-Jessica Meyer

### PR DESCRIPTION
**Pull request by jessicaleemeyer**
prep-javascript-objects-screenshot

### CODE REVIEW

File: prep-javascript-objects/prep-javascript-objects-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# prep-javascript-objects-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What data types can object properties hold?
```
Comment (BrettAlbright) [inline prep-javascript-objects/prep-javascript-objects-quiz-notes.md:7]: don't forget the quiz questions

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
just need to answer the quiz questions

File: prep-javascript-objects/prep-javascript-objects-quiz-notes.md (hunk 1 above)
Comment (jessicaleemeyer) [reply prep-javascript-objects/prep-javascript-objects-quiz-notes.md:7]: I answered the quiz questions, please confirm.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**